.github/workflows/nightly-ci.yml

```diff
@@ -0,0 +1,148 @@
+        if: always()
+        continue-on-error: true
+        with:
+            earthfile: ./catalyst-gateway/
+            flags:
+            targets: build
+            target_flags:
+            runner_address: ${{ secrets.EARTHLY_SATELLITE_ADDRESS }}
+            artifact: "false"
+
+      - name: Get schemathesis test report
+        uses: input-output-hk/catalyst-ci/actions/run@master
+        if: always()
+        continue-on-error: true
+        with:
+          earthfile: ./catalyst-gateway/tests/schemathesis_tests
+          flags: --allow-privileged
+          targets: test-fuzzer-api
+          target_flags:
+          runner_address: ${{ secrets.EARTHLY_SATELLITE_ADDRESS }}
+          artifact: "false"
+
+      - name: Get flutter unit test report
+        uses: input-output-hk/catalyst-ci/actions/run@master
+        if: always()
+        continue-on-error: true
+        with:
+            earthfile: ./catalyst_voices/
+            flags:
+            targets: test-unit
+            target_flags:
+            runner_address: ${{ secrets.EARTHLY_SATELLITE_ADDRESS }}
+            artifact: "false"
+      
+      - name: Get backend python test report
+        uses: input-output-hk/catalyst-ci/actions/run@master
+        if: always()
+        continue-on-error: true
+        with:
+            earthfile: ./catalyst-gateway/tests/api_tests/
+            flags: --allow-privileged
+            targets: test
+            target_flags:
+            runner_address: ${{ secrets.EARTHLY_SATELLITE_ADDRESS }}
+            artifact: "false"
+
+      - name: Collect and upload test reports
+        uses: actions/upload-artifact@v4
+        if: always()
+        with:
+            path: '**/*${{ env.REPORT_EXT }}'
+            if-no-files-found: error
+            retention-days: 1
+
+  generate-allure-report:
+    name: Generate allure report
+    runs-on: ubuntu-latest
+    needs: [generate-test-reports]
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/download-artifact@v4
+
+      - name: Setup Allure report
+        run: |
+              mkdir -p ${{ env.ALLURE_REPORT_PATH }}
+              shopt -s globstar
+              cp **/*${{ env.REPORT_EXT }} ${{ env.ALLURE_REPORT_PATH }}
+              ls ${{ env.ALLURE_REPORT_PATH }}
+
+      - name: Checkout gh-pages
+        uses: actions/checkout@v4
+        with:
+          ref: gh-pages
+          path: gh-pages-dir
+
+      - name: Build Allure report
+        uses: mgrybyk/allure-report-branch-action@v1
+        id: allure
+        with:
+          report_id: 'test-report'
+          gh_pages: 'gh-pages-dir'
+          report_dir: ${{ env.ALLURE_REPORT_PATH }}
+
+      - name: Git push to gh-pages
+        uses: mgrybyk/git-commit-pull-push-action@v1
+        with:
+          repository: gh-pages-dir
+          branch: gh-pages
+          pull_args: --rebase -X ours
+
+      - name: Comment PR with Allure report link
```

Comment:
You can remove this because this workflow will not be runned on a PR level

Reply:
I have removed. Alternatively, is there a way to comment Allure report link anywhere for this nightly ci?

Reply:
There is no need to. When we have the nighly working we can add the proper link [here](https://input-output-hk.github.io/catalyst-voices/architecture/10_quality/test_reports/)